content/docs/10-audit-logs/05-kube-api-audit-logging.md

```diff
@@ -0,0 +1,209 @@
+
+You can also meet internal security control requirements by enabling audit logging. Many security controls require the following capabilities.
+
+- ensuring administrators can trace the actions of individual users back to a specific person.
+
+
+- to debug an issue where an unknown application is modifying resources
+
+The guidance on this page is based on the upstream Kubernetes documentation and `kube-apiserver` source code. Follow the steps below to enable audit logging for the Kubernetes API server.
+
+<br />
+
+<WarningBox>
+
+Enabling audit logging increases the memory consumption of the API server because some context required for auditing is stored for each request. Memory consumption depends on the audit logging configuration.
+
+</WarningBox>
+
+# Prerequisites
+
+- Access to a Kubernetes cluster node.
+
+
+- Write access to the file system.
+
+
+- Remote access to the Kubernetes cluster master nodes.
+
+
+
+# Enable Auditing
+
+The Kubernetes API Audit policies contain rules about what events are captured and the level of detail to include.
+The audit policy you create will capture all requests at the *metadata* level. To learn more about the various audit levels, visit the Kubernetes API [Audit Policy](https://kubernetes.io/docs/tasks/debug/debug-cluster/audit/#audit-policy) documentation.
+
+<br />
+
+1. Log in to [Palette](https://console.spectrocloud.com).
+
+
+2. Identify one of your cluster contro-plane nodes. You find a cluster node by navigating to the left **Main Menu** and selecting **Clusters**. Click on your cluster to access the details pages and click on the **Nodes** tab. The tab contains information about each pool, select a node from the **Master Pool** to view its IP address.
+
+
+3. SSH into one of your control-plane nodes by using its IP address and the SSH key you specified during the cluster creation process. 
+
+
+
+4. From a control-plane node in the target cluster, issue the following command to create your audit policy file.
+
+  <br />
+
+  ```bash
+  cat << EOF > /etc/kubernetes/audit-policy.yaml
+    apiVersion: audit.k8s.io/v1
+    kind: Policy
+    rules:
+    - level: Metadata
+  EOF
+  ```
+  The audit log output will be written to a file located at **/var/log/kubernetes/audit/audit.log**. In production environments, you should ensure this log file is ingested by a logging and monitoring application. 
+
+  The **/var/log/kubernetes/audit/** directory should be backed by persistent storage to ensure that any unshipped audit logs will not be lost during an unexpected outage of the node.
+
+<br />
+
+5. Next, you will update the Kubernetes API server manifest file. The manifest file is located in the **/etc/kubernetes/manifests** folder. 
```

Comment:
Next, you will update the Kubernetes API server manifest file. The manifest file is in the **/etc/kubernetes/manifests** folder. 
Before you can modify the manifest file, please create a backup copy.